core/modules/ccontrol/UserQuerySelect.h

```diff
@@ -0,0 +1,147 @@
+// -*- LSST-C++ -*-
+/*
+ * LSST Data Management System
+ * Copyright 2015 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#ifndef LSST_QSERV_CCONTROL_USERQUERYSELECT_H
+#define LSST_QSERV_CCONTROL_USERQUERYSELECT_H
+/**
+  * @file
+  *
+  * @brief Umbrella container for user query state
+   *
+  * @author Daniel L. Wang, SLAC
```

Comment:
This is just a renaming of the UserQuery, I did not change much so authorship is still Daniel's. But you are right, at some point it may become difficult to say who is to blame for the mess, I think we should rely more on git history instead of sticking our names in the source.